pandas/tests/generic/test_finalize.py

```diff
@@ -177,37 +177,30 @@
         ),
         marks=not_implemented_mark,
     ),
+    pytest.param((pd.DataFrame, frame_data, operator.methodcaller("diff"))),
```

Comment:
You can remove the `pytest.param` now. It was only needed to apply the `not_implemented_mark`.